utils/unittest/fixtures.go

```diff
@@ -529,6 +529,15 @@ func ReceiptForBlockExecutorFixture(block *flow.Block, executor flow.Identifier)
 	return receipt
 }
 
+func ReceiptsForBlockFixture(block *flow.Block, ids []flow.Identifier) []*flow.ExecutionReceipt {
```

Comment:
function to return the matching ERs from different execution nodes with the same result but different execution IDs in the receipt.